docs/docs/graphql.md

```diff
@@ -283,9 +283,9 @@ query {
 }
 ```
 
-How is this possible? Via Redwood's [root schema](https://github.com/redwoodjs/redwood/blob/main/packages/api/src/makeMergedSchema/rootSchema.ts#L22-L38). The root schema is where things like currentUser are defined.
+How is this possible? Via Redwood's [root schema](https://github.com/redwoodjs/redwood/blob/4237e6a2530f55a83d683fdbf9ada887e4f4ec66/packages/graphql-server/src/rootSchema.ts#L27-L44). The root schema is where things like currentUser are defined.
```

Comment:
Ah I approved this a little too eagerly—what's the motivation for pointing to a commit instead of main? I'm guessing the answer may be something along the lines of main changes, but commits don't. But now we're pointing to old code, and still have to know when to update it anyway. (Except now it won't break in our face, so it may be harder to fix?)